packages/toolpad-app/src/theme.ts

```diff
@@ -891,11 +886,31 @@ export function getThemedComponents(theme: Theme): { components: Theme['componen
         defaultProps: {
           margin: 'dense',
         },
+        styleOverrides: {
+          root: {
+            fontSize: defaultTheme.typography.pxToRem(13),
```

Comment:
Will it be a problem that changes made for search input will affect all inputs?

Reply:
Agreed, updated

Reply:
But this is just the same but applying to all TextFields. Is there a good reason that this has to be built inside of the theme overrides? I mean, there are a lot of non-search TextFields.